packages/templates/typescript-resolvers/tests/resolvers.spec.ts

```diff
@@ -424,4 +424,148 @@ describe('Resolvers', () => {
       export type UserNameResolver<R = string | null, Parent = User, Context = any> = Resolver<R, Parent, Context>;
     `);
   });
+
+  it('should accept a map of parent types', async () => {
```

Comment:
Here it shows that fields and types get a mapper both in Result and Parent generics